EuclideanGeometry/Example/SCHAUM/Problem1.1.lean

```diff
@@ -0,0 +1,79 @@
+import EuclideanGeometry.Foundation.Index
+
+noncomputable section
+
+namespace EuclidGeom
+
+variable {P : Type _} [EuclideanPlane P]
+
+namespace Problem_1_1
+/-Let $\triangle ABC$ be an isosceles triangle in which $AB = AC$.Let $D$ be a point on $AB$.
+Let $E$ be a point on $AC$ such that $AE = AD$. Let $M$ be the midpoint of $BC$.
+
+Prove that $DM = EM$.
+-/
+--Let $\triangle ABC$ be an isosceles triangle in which $AB = AC$.
+variable {A B C : P} {hnd: ¬ colinear A B C} {hisoc: (▵ A B C).IsIsoceles}
+--Let $D$ be a point on $AB$.
+variable {D : P} {D_on_seg: D LiesInt (SEG A B)}
+--Let $E$ be a point on $AC$
+variable {E : P} {E_on_seg: E LiesInt (SEG A C)}
+--such that $AE = AD$.
+variable {E_ray_position : (SEG A E).length = (SEG A D).length}
+--Let $M$ be the midpoint of $BC$.
+variable {M : P} {median_M_position : M = (SEG B C).midpoint}
+--Prove that $DM = EM$.
+theorem Problem_1_1 : (SEG D M).length = (SEG E M).length := by
+  have h₀ : (SEG A B).length = (SEG A C).length := by
```

Comment:
Although this looks very clear, it still needs a comment.
`-- We show that the length of segment $AB$ equals to length of $AC$ `